src/transcript/TranscriptEntity.ts

```diff
@@ -0,0 +1,11 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+export default class TranscriptEntity {
+  category: string;
+  confidence: number;
+  content: string;
+  endTime: number;
+  startTime: number;
+  type?: string;
```

Comment:
Why is this only optional?

Reply:
The type filed is only present in vanilla transcribe and not in medical transcribe. Rest of the fields are present for both cases. That is why type is only kept as optional.